README.md

```diff
@@ -35,7 +35,7 @@ The `*.min.*` scripts are minified.
         new MediumEditor(document.querySelector(".editor"), {
             extensions: {
                 markdown: new MeMarkdown(function (md) {
-                    markDownEl.innerText = md;
+                    markDownEl.innerHTML = md;
```

Comment:
@connorshea Unfortunately, this is vulnerable to XSS -- while this is not a big issue, still it's important. But I will fix it!